core/modules/system/css/system.css

```diff
@@ -371,3 +371,24 @@ section,
 summary {
   display: block;
 }
+
+details {
+  cursor: text;
+}
+details > summary {
+  pointer-events: none;
+}
+details > summary a {
+  pointer-events: auto;
+}
+details > summary::after {
+  color: #0074bd;
+  cursor: pointer;
+  pointer-events: auto;
+}
+details[open] summary::after {
+  content: ' ' attr(data-close);
+}
+details:not([open]) summary::after {
+  content: ' ' attr(data-open);
+}
```

Comment:
This styling seems really specific to the specific use we want on the modules page, but we may want to use `<details>` elements in other places (such as in Form API) in the future. Can we make these selectors specific to the modules use-case and only provide generic details styling directly on the `<details>` element itself.

Reply:
All these selectors are `details..more-less-toggle` now. It won't only be used in the modules listing, but also in the status report page, the placeholder examples in the layout configuration pages, the consolidated updates page, and anywhere else we need a similar UI.

Does that address your concerns @quicksketch ?